lib/rubocop/cop/metrics/abc_size.rb

```diff
@@ -6,6 +6,7 @@ module Metrics
       # This cop checks that the ABC size of methods is not higher than the
       # configured maximum. The ABC size is based on assignments, branches
       # (method calls), and conditions. See http://c2.com/cgi/wiki?AbcMetric
+      # and https://en.wikipedia.org/wiki/ABC_Software_Metric.
```

Comment:
I guess this links can also go to the references metadata.

Reply:
Will do. This is the only cop that now has more than one reference. We may want to readdress the way we handle references in the future.